test/integration/matchers/matcher-utils.ts

```diff
@@ -202,23 +203,26 @@ function toBpmnStyle(rawStyle: StyleMap, isEdge: boolean): BpmnCellStyle {
     markers: rawStyle[BpmnStyleIdentifier.MARKERS]?.split(',').sort(),
     // for message flow icon (value in rawStyle are string)
     'bpmn.isInitiating': rawStyle[BpmnStyleIdentifier.IS_INITIATING] ? rawStyle[BpmnStyleIdentifier.IS_INITIATING] == 'true' : undefined,
-  };
 
-  if (isEdge) {
-    style.startArrow = rawStyle.startArrow;
-    style.endArrow = rawStyle.endArrow;
-    style.endSize = rawStyle.endSize;
-  } else {
-    style.shape = rawStyle.shape;
-    style.horizontal = rawStyle.horizontal;
-    style.swimlaneFillColor = rawStyle.swimlaneFillColor;
-    style.fillOpacity = rawStyle.fillOpacity;
-    style.gradientColor = rawStyle.gradientColor;
-    style.gradientDirection = rawStyle.gradientDirection;
-  }
-  return style;
+    ...(isEdge
+      ? {
+          startArrow: rawStyle.startArrow,
+          endArrow: rawStyle.endArrow,
+          endSize: rawStyle.endSize,
+        }
+      : {
+          shape: rawStyle.shape,
+          horizontal: rawStyle.horizontal,
+          swimlaneFillColor: rawStyle.swimlaneFillColor,
+          fillOpacity: rawStyle.fillOpacity,
+          gradientColor: rawStyle.gradientColor,
+          gradientDirection: rawStyle.gradientDirection,
+        }),
+  };
 }
+/* eslint-enable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
 
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
**nitpick:** no-unsafe-assignment has just been enabled above. If there is a particular reason, please make it explicit in the comment